src/pudl_archiver/depositors/zenodo.py

```diff
@@ -156,13 +156,21 @@ async def get_deposition(self, concept_doi: str) -> Deposition:
             )
 
         latest_deposition = Deposition(**sorted(response, key=lambda d: d["id"])[-1])
-        full_deposition_json = await self.request(
+        return await self.get_record(latest_deposition.id_)
+
+    async def get_record(self, rec_id: int) -> Deposition:
```

Comment:
The direct-access endpoint seems to be more reliably up-to-date than the conceptdoi query.

Reply:
This definitely seems like a good improvement